release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1717,6 +1717,11 @@ The following related base images for Operator projects are _not_ deprecated. Th
 
 For information about the unsupported, community-maintained, version of the Operator SDK, see link:https://sdk.operatorframework.io[Operator SDK (Operator Framework)].
 
+[id="ocp-4-16-deprecation-preserveBootstrapIgnition{context}"]
```

Comment:
```suggestion
[id="ocp-4-16-deprecation-preserveBootstrapIgnition_{context}"]
```
need an underscore between text and context tag

Reply:
:heavy_check_mark: